ingestion/src/metadata/ingestion/source/dashboard/tableau/metadata.py

```diff
@@ -375,15 +383,27 @@ def yield_dashboard_chart(
                 if filter_by_chart(self.source_config.chartFilterPattern, chart.name):
                     self.status.filter(chart.name, "Chart Pattern not allowed")
                     continue
-                workbook_name = dashboard_details.name.replace(" ", "")
                 site_url = (
                     f"site/{self.service_connection.siteUrl}/"
                     if self.service_connection.siteUrl
                     else ""
                 )
+                workbook_chart_name: ChartUrl = (
+                    ChartUrl(
+                        workbook_name=chart.content_url.split("/")[0],
+                        sheets=chart.content_url.split("/")[1],
+                        chart_url_name=chart.content_url.split("/")[2],
```

Comment:
nit - to clarify this spli:

```python
workbook_name, sheets, chart_url_name, *_ = chart.content_url.split("/")
```

instead of splitting 3 times and getting each element